### DISCUSSION TITLE
why does resize not affect affine?

### DISCUSSION THREAD

**Discussion by function2-llx** [Project-MONAI/MONAI#3480] [Q&A] [answered]
Hello MONAI community and dev group,

When I'm using transforms from MONAI, I notice that when I resize an image, its affine does not change. This is inconsistent with my understanding of affine in NIfTI (according to [a page from 3D Slicer](https://www.slicer.org/wiki/Coordinate_systems)), because I think when the image is resized, the transformation from the image coordinate system to the anatomical coordinate system should change as well. I'd appreciate it if anyone could explain this. 

Thanks in advance.

**Comment by Nic-Ma** [comment, marked as answer]
Hi @function2-llx ,

Thanks for raising this ticket.
You are right, currently, we don't update the affine in transforms except `Spacing` and `Orientation`.
We are already working on it to add meta data updating in all the transforms, suppose to release in MONAI 1.0.
CC @wyli @Spenhouet 

Thanks
